components/dialogs/ResourceUploadDialog.tsx

```
@@ -185,12 +203,23 @@ export function ResourceUploadDialog({
             formData.append("file", file);
 
             try {
-              return await axios.post("/api/resources/upload", formData, {
+              // Add sessionId to URL if available
+              let url = "/api/resources/upload";
+              if (sessionId) {
+                url += `?sessionId=${sessionId}`;
+                console.log(
+                  "Using sessionId for upload:",
+                  sessionId.substring(0, 8) + "..."
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)